Documentation/Signoffs/php-communication-signoff.md

```diff
@@ -0,0 +1,68 @@
+# Communication Module Signoff
+
+## Subsystem Function
+The function of the communication subsystem is to act as a transmitter for each of the sensor modules and a receiver for the head unit. The function of the transmitter is to send the sensor name, data type, sensor number, and sensor data. The function of the receiver is to receive all of the data from the sensor modules and make the received information available for the processor. 
+
+![TransmitterCommunication](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/142913669/cfb66cac-4edb-49bc-8843-ddd06d3072f7)
+
+![RecieverCommunication](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/142913669/8ede150c-271b-4bfd-8071-dab157138e04)
+
+
+## Constraints
+| No. | Constraints                                                                                    | Origin                              |
+| --- | ---------------------------------------------------------------------------------------------- | ----------------------------------- |
+|  1  | Sensors shall not be placed in a way where they cannot wirelessly communicate with the system. | Project Team & Broader Implications |
+|  2  | System shall not require an internet connection to work and communicate with the head unit and sensors | Home Owners, Insurance Agencies, & Team Supervisor |
+|  3  | Sensor communication shall send the sensor name, number, data type, and raw data to the head unit. | Project Team |
+|  4  | Sensor modules shall not send more than 64 bits of data when transmitting information to the head unit. | Project Team & Standards|
```

Comment:
why?

Reply:
> why?

We explained why the sensors shall not send more than 64 bits of data underneath the constraints section on number 6.